### ISSUE TITLE
 Report the number of lines in a text file

### ISSUE THREAD

**Issue by pombredanne** [nexB/scancode-toolkit#1350]
Since scans (license and copyrights) typically return line ranges, it would be useful to return the number of lines if a file is a text file.

**Comment by sritasngh** [comment]
@pombredanne scans (license and copyrights) return line ranges for the text matched. When you say to include number of lines do you mean to add it for the whole file like [here](https://gist.github.com/itssingh/4eee9bc42fadf1b86e6c533767119080#file-fsolver_output-json-L37)?

**Comment by pombredanne** [comment]
> When you say to include number of lines do you mean to add it for the whole file like here?

yes, and we have some code to port and start from  for this here https://github.com/nexB/scancode-toolkit-contrib/blob/develop/src/sourcecode/metrics.py

**Comment by Siddhant-K-code** [comment]
@itssingh will you like to work on it? or I can work upon it.

**Comment by sritasngh** [comment]
> @itssingh will you like to work on it? or I can work upon it.

working on it:)
